WordPress/src/zbetagroup/AndroidManifest.xml

```diff
@@ -1,28 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          xmlns:tools="http://schemas.android.com/tools"
-          package="org.wordpress.android"
-          android:installLocation="auto">
-
-    <!-- GCM configuration -->
-    <permission
-        android:name="org.wordpress.android.beta.permission.C2D_MESSAGE"
-        android:protectionLevel="signature"/>
-
-    <uses-permission android:name="org.wordpress.android.beta.permission.C2D_MESSAGE"/>
-
-    <application>
-
-        <!-- We've used a custom receiver here. The one that is bundled in GCM API wasn't able to start the service. -->
```

Comment:
I read the documentation again, I got confused because their sample app doesn't reflect it: ([AndroidManifest](https://github.com/googlesamples/google-services/blob/master/android/gcm/app/src/main/AndroidManifest.xml))

The doc says `C2D_MESSAGE` perm is only needed to prevent other apps to hijack PNs - I'll add that again, but I don't think we need a specific case for release or beta.